iothub/device/src/IotHubModuleClient.cs

```diff
@@ -267,23 +266,22 @@ private async Task<DirectMethodResponse> InvokeMethodAsync(Uri uri, DirectMethod
 
             try
             {
-                var transportSettings = new IotHubClientHttpSettings();
+                var pipelineContext = new PipelineContext
+                {
+                    IotHubConnectionCredentials = IotHubConnectionCredentials,
+                    HttpOperationTransportSettings = new IotHubClientHttpSettings(),
+                };
 
                 if (customCertificateValidation != null)
                 {
                     httpClientHandler = new HttpClientHandler
                     {
                         ServerCertificateCustomValidationCallback = customCertificateValidation,
-                        SslProtocols = transportSettings.SslProtocols,
+                        SslProtocols = pipelineContext.HttpOperationTransportSettings.SslProtocols,
                     };
                 }
 
-                var pipelineContext = new PipelineContext
-                {
-                    IotHubConnectionCredentials = IotHubConnectionCredentials,
-                };
-
-                using var httpTransport = new HttpTransportHandler(pipelineContext, transportSettings, httpClientHandler);
+                using var httpTransport = new HttpTransportHandler(pipelineContext, httpClientHandler);
```

Comment:
This should hopefully be refactored out to not require a separate pipeline context instance once we have integrated http into the pipeline flow. (This might be similar to the file upload flow we have on the device client).